### ISSUE TITLE
Compose multiplatform

### ISSUE THREAD

**Issue by mahramane** [mr0xf00/easycrop#11]
Hi
There isn't any image cropper library for compose multiplatform (jvm, android, ios, ...).
You created base of compose code and just need to change some code for compose multiplatform.
It will be very useful and i really need it.
Please work on it 🙏.

**Comment by mahramane** [comment]
...

**Comment by JustAmalll** [comment]
++

**Comment by tamimattafi** [comment]
@mahramane @JustAmalll I started implementing CMP in my fork, I will create some PRs. The author seems to be absent, so I will try to publish the changes to my maven central account.

Here's the first PR if you wish to upvote: https://github.com/mr0xf00/easycrop/pull/12
You can follow my fork for further news.

**Comment by mahramane** [comment]
> @mahramane @JustAmalll I started implementing CMP in my fork, I will create some PRs. The author seems to be absent, so I will try to publish the changes to my maven central account.
> 
> Here's the first PR if you wish to upvote: https://github.com/mr0xf00/easycrop/pull/12
> You can follow my fork for further news.

Thank you 🙏🏻.

**Comment by tamimattafi** [comment]
@mahramane @JustAmalll I updated the pull request, CMP support is already there
https://github.com/mr0xf00/easycrop/pull/12

**Comment by mahramane** [comment]
> @mahramane @JustAmalll I updated the pull request, CMP support is already there #12

very good 👍👍
thank you
Does it support desktop and wasmjs?

**Comment by tamimattafi** [comment]
@mahramane Yes, it does support these targets, however, I only tested on Android and iOS (Using the sample app)
Edit: here are the [supported targets](https://github.com/tamimattafi/krop/blob/main/plugins/multiplatform/src/main/kotlin/com/mr0xf00/easycrop/multiplatform/MultiplatformConventions.kt)

**Comment by tamimattafi** [comment]
For anyone looking for a multiplatform version, you can get it here: https://github.com/tamimattafi/krop
